packages/synthetic-chain/public/upgrade-test-scripts/run_prepare_zero.sh

```diff
@@ -12,8 +12,17 @@ export CHAINID=agoriclocal
 agd init localnet --chain-id "$CHAINID"
 
 allaccounts=("gov1" "gov2" "gov3" "user1" "validator")
-for i in "${allaccounts[@]}"; do
-  cat "/usr/src/upgrade-test-scripts/keys_for_test_only/$i.key" | agd keys add $i --recover --keyring-backend=test 2>&1
+# WARNING: these mnemonics are purely for testing purposes, do not implement
+# features that depend on them in any way.
+allkeys=(
```

Comment:
may want to point out that the names come from matching indices in `allaccounts`

Reply:
Good call, added a comment above